packages/lab/src/date-picker/DatePickerRangePanel.tsx

```diff
@@ -352,12 +354,12 @@ export const DatePickerRangePanel = forwardRef<
           <Calendar selectionVariant={"range"} {...StartCalendarProps}>
```

Comment:
What's the best way to do single calendar for a range picker?

Reply:
it's possible, should be as simple as adding a prop to the panel that specifies the number of calendars in the view
Then you need to do something about navigation, again should be simple, as long as rendered calendars are all based of the visible month. Will revisit in another PR/example